dashboard/src/components/NotFound/404.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
We should probably actually put this in https://github.com/kubeapps/kubeapps/tree/master/dashboard/src/img.